storybook/src/components/Radio/Radio.docs.mdx

```diff
@@ -35,3 +35,12 @@ If the Radio can become invalid, add an Error Message component and its `id` to
 Check [the Field Set docs](/docs/components-forms-field-set--docs) for more information on configuring it.
 
 <Canvas of={RadioStories.InAFieldSetWithValidation} />
+
+### Custom icons
+
+It is possible to change the default Radio icon.
+This is intended for users who do not work within the City of Amsterdam corporate identity.
+If you work for the City of Amsterdam, it is unlikely that you need this.
+If you would like to use a custom icon, please contact designsystem@amsterdam.nl.
```

Comment:
```suggestion
If you would like to use a custom icon, please contact [designsystem@amsterdam.nl](mailto:designsystem@amsterdam.nl).
```
You could link it but also, maybe not (spam) if you agree on the other comments in the custom icons story about the svg class named inner/outer stuff. So you add `<svg>` element example in there that is selectable. Don't know if taking these kinds of requests is a good idea, the documentation should be enough.

Reply:
Linking explicitly isn't necessary, MDX already does that for you.

Using a custom icon is still kind fiddly, regardless of the approach we use. We could explain it in detail in the docs, but that feels like overkill for something that might never be used? It's kind of a fringe feature imo

Reply:
I’ve updated the copy and removed the suggestion to send an e-mail.